packages/cf.js/src/utils/uuid.ts

```diff
@@ -0,0 +1,60 @@
+/* tslint:disable */
+const rng = crypto.getRandomValues(new Uint8Array(16));
```

Comment:
We assume `crypto` is in scope?

Is it in scope in the browser?

Reply:
Supported in modern browsers: https://caniuse.com/#feat=getrandomvalues